pom.xml

```diff
@@ -230,51 +258,150 @@
         <module>openidm-util</module>
         <module>openidm-workflow-activiti</module>
         <module>openidm-maintenance</module>
-        <!--module>openidm-workflow-remote</module-->
         <module>openidm-zip</module>
         <module>custom-scripted-connector-bundler</module>
     </modules>
 
-    <profiles>
-        <profile>
-            <id>jrebel</id>
+    <build>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.forgerock.commons</groupId>
+                    <artifactId>i18n-maven-plugin</artifactId>
+                    <version>${i18nPluginVersion}</version>
+                    <executions>
+                        <execution>
+                            <phase>generate-sources</phase>
 
-            <build>
-                <plugins>
-                    <plugin>
-                        <groupId>org.zeroturnaround</groupId>
-                        <artifactId>jrebel-maven-plugin</artifactId>
-                        <version>1.1.3</version>
+                            <goals>
+                                <goal>generate-messages</goal>
+                            </goals>
+
+                            <configuration>
+                                <messageFiles>
+                                    <messageFile>com/example/my_tool.properties</messageFile>
+                                </messageFiles>
+                            </configuration>
+                        </execution>
+                    </executions>
+                </plugin>
 
-                        <configuration>
-                            <packaging>jar</packaging>
-                        </configuration>
+                <plugin>
+                    <groupId>org.codehaus.mojo</groupId>
+                    <artifactId>jslint-maven-plugin</artifactId>
+                    <version>${jslintPluginVersion}</version>
+                    <configuration>
+                        <assumeConsoleAlertEtc>true</assumeConsoleAlertEtc>
+                        <strictWhiteSpace>false</strictWhiteSpace>
+                    </configuration>
+                </plugin>
 
-                        <executions>
-                            <execution>
-                                <id>generate-rebel-xml</id>
-                                <phase>process-resources</phase>
-                                <goals>
-                                    <goal>generate</goal>
-                                </goals>
-                            </execution>
-                        </executions>
-                    </plugin>
+                <!--This plugin's configuration is used to store Eclipse m2e settings only. It has no influence on the Maven build itself. -->
+                <plugin>
+                    <groupId>org.eclipse.m2e</groupId>
+                    <artifactId>lifecycle-mapping</artifactId>
+                    <version>1.0.0</version>
```

Comment:
I am all for replacing this plugin with processing instructions (if possible):

* https://www.eclipse.org/m2e/documentation/release-notes-17.html#new-syntax-for-specifying-lifecycle-mapping-metadata
* https://github.com/WrenSecurity/wrensec-parent/blob/master/pom.xml#L1026

Reply:
Plugin configuration has been completely removed.